src/movies.js

```diff
@@ -1,19 +1,66 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    let directores = moviesArray.map ((directors) => {
+        return directors.director
+    })
+    
+return directores}
+
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+    let i = 0
+    stiPeli = moviesArray.filter ((numSpielberg) => {
+        if(numSpielberg.director === "Steven Spielberg"){
+           for(i=0; i < numSpielberg.genre.length; i++){
+            if (numSpielberg.genre[i] == "Drama"){
+               let total = 0
```

Comment:
`let total = 0 `debe definirse en el scope global, es decir afuera del filter para poder regresar el resultado afuera del filter.
Dentro del for se realiza la suma para ir agregando cada elemento ej. `total ++` o `total += 1`.